pkg/sql/schemachanger/scdecomp/testdata/table

```diff
@@ -11,6 +11,7 @@ COMMENT ON TABLE tbl IS 'tbl is good table';
 COMMENT ON INDEX tbl@tbl_pkey IS 'tbl_pkey is a primary key';
 COMMENT ON COLUMN tbl.id IS 'id is a identifier';
 COMMENT ON CONSTRAINT myfk ON tbl IS 'must have a parent';
+ALTER TABLE tbl CONFIGURE ZONE USING gc.ttlseconds=10;
```

Comment:
nit: can you add a case (or replace this one) where you add a zone config to an index instead of the table itself?